Redo header to be closer to vererans.gc.ca

smcmurtry opened the pull request:
Closes #1959

smcmurtry committed:
wip

smcmurtry committed:
wip

smcmurtry committed:
got the styling working

smcmurtry committed:
added french components, tests

smcmurtry committed:
deleted unused component

smcmurtry committed:
removed unused prop

smcmurtry committed:
removed unused prop from test

smcmurtry committed:
empty commit

brdunfield commented:
It looks like VAC's header has a bit more padding on the bottom, as well as slightly different spacing between the icons on the right (and on mobile).

smcmurtry committed:
wip

smcmurtry committed:
merged in master

smcmurtry committed:
wip

smcmurtry committed:
added some missing styles and nbsp

smcmurtry commented:
@brdunfield I added some padding and I've been trying to debug the styling differences for the last couple hours. I added some missing styles, and as far as I can tell, the computed css for this component is now the same as on veterans.gc.ca. I also added a non breaking space to the My VAC link. It's still not a perfect match but I think this is the best I can do.

smcmurtry committed:
empty

brdunfield reviewed (approved):
Looking good! 🦑
